lxd/instance/instance_utils.go

```diff
@@ -1064,7 +1069,12 @@ func MoveTemporaryName(inst Instance) (string, error) {
 		}
 	}
 
-	return fmt.Sprintf("lxd-move-of-%s", instUUID), nil
+	return CreateTemporaryName(instUUID), nil
+}
+
+// IsTemporaryName returns true if the given instName equals the temporary name derived from otherInstUUID.
+func IsSameTemporaryInstance(instName string, otherInstUUID string) bool {
```

Comment:
No need for this function, it just performs an `==` on 2 strings.

Reply:
Removed.